spec/src/main/asciidoc/repository.asciidoc

```diff
@@ -839,44 +1015,6 @@ first20 = products.findByNameLike(name, pageable);
 
 ----
 
-=== Methods With Entity Parameters
-
-Repository methods with a name that begins with one of the prefixes, `save` or `delete`, can have a single entity parameter that is one of the following types, where `E` is the entity type:
-
-- `E` - the entity type
-- `E[]` - an array of the entity type
-- `E...` - a variable arguments array of the entity type
-- `Iterable<E>` and subtypes of `Iterable<E>` - a collection of multiple entities
-
-Note: A form of `delete` can be defined in a different manner under the Query by Parameters and Query by Method Name patterns. In those cases, the method can have multiple parameters, none of which can be the entity type.
-
-==== Save Methods
-
-Save methods are a combination of update and insert where entities that are already present in the database are updated and entities that are not present in the database are inserted.
-
-The unique identifier is used to determine if an entity exists in the database. If the entity exists in the database and the entity is versioned (for example, with `@code jakarta.persistence.Version` or by another convention from the entity model such as having an attribute named `version`), then the version must also match. When updates are saved to the database, the version is automatically incremented. If the version does not match, the `save` method must raise `OptimisticLockingFailureException`.
-
-A `save` method parameter that supplies multiple entities might end up updating some and inserting others in the database.
-
-===== Generated Values
-
-When saving to the database, some entity attributes might be automatically generated or automatically incremented in the database. To obtain these values, the user can define the `save` method to return the entity type or a type that is a collection or array of the entity. Entities that are returned by `save` methods must include updates that were made to the entity. Jakarta Data does not require updating instances of entities that are supplied as parameters to the method.
-
-Example usage:
-
-[source,java]
-----
-product.setPrice(15.99);
-product = products.save(product);
-System.out.println("Saved version " + product.getVersion() + " of " + product);
-----
-
-==== Delete Methods
-
-Delete methods remove entities from the database based on the unique identifier of the entity parameter value. If the entity is versioned (for example, with `@code jakarta.persistence.Version` or by another convention from the entity model such as having an attribute named `version`), then the version must also match. Other entity attributes do not need to match. The the unique identifier of an entity is not found in the database or its version does not match, a `delete` method with a return type of `void` or `Void` must raise `OptimisticLockingFailureException`.
-
-==== Return Types
-
 Refer to the Jakarta Data module JavaDoc section on "Return Types for Repository Methods" for a listing of valid return types for methods with entity parameters.
```

Comment:
Most of the detail in this section with the specification requirements for delete(entity) and save(entity) operations was lost in your rewrite.  The easiest fix would be to put the whole section back.

(FYI - This is where I have a separate pull that has been held up that will add update/insert requirements as well).

Reply:
It no longer matters that the sections on Delete and Save were lost because the requirements are now documented on the annotations themselves (in the JavaDoc), so the issue I originally raised here no longer applies and I'll mark this conversation as resolved.